### ISSUE TITLE
Add twisted/trial/__init__.py to dist3.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8541]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8541|
|Type|enhancement|
|Created|2016-06-30 03:45:15Z|
|Branch|https://github.com/twisted/twisted/tree/8541-rodrigc-trial|

It looks like in previous Python 3 porting work, twisted/trial/__init__.py was omitted from dist3.py.
This file is needed to get more of the trial tests running under Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8541 8541
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8541_rodrigc_trial 8541-rodrigc-trial
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467258315840530 1467258315840530
changetime__1467271271207972 1467271271207972
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/275

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Looks good to me, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 44deebfc6a8fe52fd40d3d35ecce585fc4151531
```
#!CommitTicketReference repository="" revision="44deebfc6a8fe52fd40d3d35ecce585fc4151531"
Merge 8541-rodrigc-trial: Add twisted/trial/__init__.py to dist3.py to enable more trial tests on Python 3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8541
```
